docs/source/usersguide/basics.rst

```diff
@@ -0,0 +1,213 @@
+which stands for eXtensible Markup Language, is a simple format that allows data
+to be exchanged efficiently between different programs and interfaces.
+
+Anyone who has ever seen webpages written in HTML will be familiar with the
+structure of XML whereby "tags" enclosed in angle brackets denote that a
+particular piece of data will follow. Let us examine the follow example:
+
+.. code-block:: xml
+
+    <person>
+      <firstname>John</firstname>
+      <lastname>Smith</lastname>
+      <age>27</age>
+      <occupation>Health Physicist</occupation>
+    </person>
+
+Here we see that the first tag indicates that the following data will describe a
+person. The nested tags *firstname*, *lastname*, *age*, and *occupation*
+indicate characteristics about the person being described.
+
+In much the same way, OpenMC input uses XML tags to describe the geometry, the
+materials, and settings for a Monte Carlo simulation. Note that because the XML
+files have a well-defined structure, they can be validated using the
+:ref:`scripts_validate` script.
+
+.. _XML: http://www.w3.org/XML/
+
+Creating Input Files
+--------------------
+
+.. currentmodule:: openmc
+
+The simplest option to create input files is to simply write them from scratch
+using the :ref:`XML format specifications <io_file_formats_input>`. This
+approach will feel familiar to users of other Monte Carlo codes such as MCNP and
+Serpent, with the added bonus that the XML formats feel much more
+"readable". Alternatively, input files can be generated using OpenMC's
+:ref:`Python API <pythonapi>`, which is introduced in the following section.
+
+----------
+Python API
+----------
+
+OpenMC's :ref:`Python API <pythonapi>` defines a set of functions and classes
+that roughly correspond to elements in the XML files. For example, the
+:class:`openmc.Cell` Python class directly corresponds to the
+:ref:`cell_element` in XML. Each XML file itself also has a corresponding class:
+:class:`openmc.Geometry` for ``geometry.xml``, :class:`openmc.Materials` for
+``materials.xml``, :class:`openmc.Settings` for ``settings.xml``, and so on. To
+create a model then, one creates instances of these classes and then uses the
+``export_to_xml()`` method, e.g. :meth:`Geometry.export_to_xml`. Most scripts
+that generate a full model will look something like the following:
+
+.. code-block:: Python
+
+   # Create materials
+   materials = openmc.Materials()
+   ...
+   materials.export_to_xml()
+
+   # Create geometry
+   geom = openmc.Geometry()
+   ...
+   geom.export_to_xml()
+
+   # Assign simulation settings
+   settings = openmc.Settings()
+   ...
+   settings.export_to_xml()
+
+One a model has been created and exported to XML, a simulation can be run either
+by calling :ref:`scripts_openmc` directly from a shell or by using the
+:func:`openmc.run()` function from Python.
+
+If you have never used Python before, the prospect of learning a new code *and*
```

Comment:
I would suggest moving this paragraph, and the bulleted list and tip block below, to the top of the PyAPI section.